src/ds/json_schema.h

```diff
@@ -51,6 +51,9 @@ namespace ds
     template <typename T>
     void fill_schema(nlohmann::json& schema);
 
+    template <typename T>
```

Comment:
needed here, same as `fill_schema`, otherwise we will get `undeclared identifier` compiler error since using `std::array` for storing the `digest_sig` in `pbftpreprepares.h` will call `fill_json_schema`